src/components/expression/expression-renderer.js

```diff
@@ -968,13 +1118,40 @@ genotet.ExpressionRenderer.prototype.highlightLabelsAfterUpdateData_ =
       this.clickedObject_.column = heatmapData.conditionNames.indexOf(
         this.clickedObject_.conditionName
       );
-      this.signal('expressionClick', this.clickedObject_);
+    }
+    if (this.clickedObject_.row != -1 && this.clickedObject_.column != -1) {
```

Comment:
This kind of comparison is very common in many files... If we give constant here, we should give constant in other files too. I don't think it is that necessary.